integration_tests/test_coverage.py

```diff
@@ -51,6 +51,9 @@ def _read_test_config():
     assert "exclude_path" in coverage_config
     assert "crate_features" in coverage_config
 
+    assert coverage_config["crate_features"].find(' ') == -1, \
```

Comment:
nit: I find the syntax `' ' not in coverage_config["crate_features"]` to be more descriptive than using `-1` for the check.

Reply:
Good point. Updated that.